test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -104,6 +104,19 @@
                                               :fi "NBN fi"
                                               :sv "NBN sv"}}
                               :owneruserid "owner"} :en counts)))
+      (is (= "y ([:t.administration/org]: NBN en) ([:t.administration/licenses]: Performance License, Second License)"
+             (resource-label {:resid "y"
+                              :licenses [{:localizations {:en {:title "Performance License"}
+                                                          :fi {:title "Performance License FI"}
+                                                          :sv {:title "Performance License SV"}}}
+                                         {:localizations {:en {:title "Second License"}
+                                                          :fi {:title "Second License FI"}
+                                                          :sv {:title "Second License SV"}}}]
+                              :organization {:organization/id "nbn"
+                                             :organization/short-name {:en "NBN en"
+                                                                       :fi "NBN fi"
+                                                                       :sv "NBN sv"}}} :en counts)))
```

Comment:
I like an explicit test like this. you could also have a literal `{"y" 2}` for `counts`. Then you could only have literal cases like this, and wouldn't need the `let` at the start of this test at all.